common/static/js/utils/edx.utils.validate.js

```diff
@@ -1,186 +1,187 @@
-var edx = edx || {};
-
-(function( $, _, _s, gettext ) {
+;(function (define) {
```

Comment:
Since you're changing this class, I'd recommend also moving it to``common/static/common/js`` which is where we want all common code to live. The idea is that this is namespaced and so it can be safely used in other contexts. I'd probably name it:

    common/static/common/js/utils/validate.js

@symbolist I'm not sure if you're aware that we now have two very similar JavaScript libraries for building client-side forms. There's the one for logistration, and then also the one used by account settings. In the long run we should refactor these into a single framework.